index.css

```diff
@@ -0,0 +1,68 @@
+body{
+  margin: 0;
+}
+/*start page css*/
+.page{
+  width: 495px;
+  margin: auto;
+  text-align: center;
+}
+.page .header{
+  height: 50px;
+  border: 1px solid red;
+  padding: 2px 0 2px 20px;
+}
+.page .content{
+  border: 1px solid red;
+  min-height: 500px;
+}
+.page .footer{
+  height: 50px;
+  border: 1px solid red;
+}
+/*end page css*/
+
+/*start header css*/
+.header .header-left{
+  float: left;
+}
+.header .header-right{
+  float: right;
+}
+/*end header css*/
+
+/*start navbar css*/
+.navbar{
```

Comment:
in the app, we can have more than 1 navbar, so this can make other confuse about the css for which navbar.
It will be nicer if we change the name to main-menu or some other specifically name.
As we only have 1 main-menu in the app.